flexmeasures/data/services/sensors.py

```diff
@@ -1,7 +1,8 @@
 from __future__ import annotations
 
 from datetime import datetime, timedelta
-from timely_beliefs import BeliefsDataFrame
+from pandas.core.groupby.generic import DataFrameGroupBy
+from typing import Dict, List
```

Comment:
Not required. You can do `-> list[dict]` instead of `List[Dict]`. We use `from __future__ import annotations` for compatibility with Python 3.8 (and 3.9?).